### PR TITLE
[ZEPPELIN-5184] Update related to entity name of credentials

### PR DESCRIPTION
**Pull request by comma337**
### What is this PR for?
The entity name of the credential has changed.
Please refer to these. https://github.com/apache/zeppelin/pull/3806, https://github.com/apache/zeppelin/pull/4008.
The documentation and web UI need to be updated.
Change the entity name from [Interpreter Group].[Interpreter Name] to [Interpreter Name].

### What type of PR is it?
[Improvement | Documentation]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-5184

### How should this be tested?
Access the web page.

#### Web UI
- http://[zeppelin-server]:[zeppelin-port]/#/credential
- Change the entity name from [Interpreter Group].[Interpreter Name] to [Interpreter Name].
- default UI
  - before 
    default_before1
    default_before2
  - after
    default_after1
    default_after2
- next UI
  - before
    nextui_before
  - after
    nextui_after

#### Documentation
- http://[zeppelin-doc-server]:[zeppelin-doc-port]/setup/security/datasource_authorization.html#how-to-save-the-credential-information
- Change the entity name from [Interpreter Group].[Interpreter Name] to [Interpreter Name].
  - before
    doc_before1
  - after
    doc_after1

- http://[zeppelin-doc-server]:[zeppelin-doc-port]/usage/interpreter/overview.html#credential-injection
- Fix broken image
  - before
    doc_before2
  - after
    doc_after2

### Questions:
* Does the licenses files need update? N
* Is there breaking changes for older versions? N
* Does this needs documentation? Y

### CODE REVIEW

File: docs/_config.yml [hunk 1]
```diff
@@ -21,11 +21,7 @@ author :
   twitter : ASF
   feedburner : feedname
 
-<<<<<<< Updated upstream
-ZEPPELIN_VERSION : 0.9.0-SNAPSHOT
-=======
 ZEPPELIN_VERSION : 0.10.0-SNAPSHOT
->>>>>>> Stashed changes
```
Comment (zjffdu) [inline docs/_config.yml:28]: Thanks for fixing these conflicts as well

**Comment by zjffdu** [comment]
LGTM

**Comment by zjffdu** [comment]
Will merge if no more comment

**Comment by comma337** [comment]
@zjffdu Thank your for your help!

**Comment by zjffdu** [comment]
> @zjffdu Thank your for your help!

No problem, welcome more contributions to zeppelin

**Pull request closed without merging**